config/oauth2_config.xml.sample

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<OAuth2.0>
+    <provider name="Google">
+        <!-- Keep the client_secret file on a path inaccessible to public.  -->
+        <client_secret_file>./oauth2/client_secret.json</client_secret_file>
```

Comment:
secrets contained in these config files are (unless something is very wrong) inaccessible to the public. I would rather enter my client ID / client secret in this XML file rather than in a separate json file. Compare with `bind-dn`/`bind-password` from ldap's configuration. https://github.com/galaxyproject/galaxy/blob/dev/config/auth_conf.xml.sample#L94